### DISCUSSION TITLE
2022-05-25 Weekly Meeting 57 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#236] [Weekly Meetings]
Weekly Meeting 57 is happening on Wednesday May 25th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
Any demos or things to share from the community? 

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
  - blog update
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - First developer hour! 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
- [Media Group](https://github.com/omigroup/media)
- any others?

**Comment by madjin** [comment]
I can talk some about buildathon plans and interest groups interested in supporting such

**Comment by lyuma** [comment]
We're going to catch up for 20 minutes with the agenda items, then open it up for the last 30-40 minutes for demos and other topics.

**Comment by madjin** [comment]
## Notes
- humbletim observing the ecosystem around blender
- Anthony is working on blog enablement to turn on the blog tags, editorial path, etc
- Working on 1 year anniversary post, updates on working groups / info about them
  -  https://omigroup.org/?p=374&preview=true (you'll need to be logged in as your user)
- Scripting group talked about graphics stuff like LODs, imposters, scaling
- Next scripting group meetup (Tuesday 11:30am EST) will be demo day
  - Adam will show off scripting pipeline for Sinespace
  - Group is early, lot of open discussion, good opportunity to show off cool WIP

**Comment by madjin** [comment]
glTF working group: https://github.com/omigroup/gltf-extensions/discussions

- reconcile audio extension with khronos khr audio
- working on physics colliders
- talked about tooling
 - difficult to make gltf content with various extensions out there
 - how to build better authoring tools
 - OMI_ref: embed gltf documents inside other glTF documents
 - missing element to compose gltf scenes inside each other
 - can use in editor, dynamic gltfs with other things happening in scene
 - https://github.com/omigroup/gltf-extensions/pull/85

**Comment by madjin** [comment]
Evo has been testing sticker designs for holographic stickers to print for laptops, etc = looking at $150 + mail costs to make stickers for our community (a run of 200 to start, could be more). We have budget for this but wanted to hear thoughts before we order these gifts. Evo is down to tweak designs in Canva with others if anyone else wants to contribute in any way.
![image](https://user-images.githubusercontent.com/32600939/170309018-5ac290f8-527e-405e-9345-6d3491ab5c8e.png)

**Comment by madjin** [comment]
## OMI Media

- If wanna get involved join the discord channel / dig into the github (https://github.com/omigroup/media)
- can participate in various channels (twitter/twitch/mastodon/reddit/youtube/etc)
- can submit media here: https://airtable.com/shrZAsV1MA5QJctYx

**Comment by madjin** [comment]
Evo got access to physical hackerspace called circuit launch, near oakland airport: https://circuitlaunch.com/
